partials/mixins/panel.html

```diff
@@ -0,0 +1,5 @@
+<div id="{{toggle}}-panel" class="reveal js-hidden">
+    <div class="panel-indent{{#getErrorClass}}{{toggle}}{{/getErrorClass}}">
```

Comment:
Ours look like:
![other-title-cropped](https://cloud.githubusercontent.com/assets/3900826/15500551/179bd220-21a1-11e6-8357-75796845d9fb.png)

Config may be the route to go but consistency in style would also be good.